utils/lists.py

```diff
@@ -1,6 +1,15 @@
+from utils.trivia_data import QuestionAnswer, QuestionList
+
 ballresponse = [
```

Comment:
I agree a comment would be useful, but I wonder if it should be a separate PR. `ballresponse`, as explained to Max above, is pre-existing code that I only touched because my linter was yelling at me over the original author's non-standard indentation and I couldn't quickly find the option to silence that alert.

I might be able to make a case for including such a comment in this PR since I'm touching that code anyway, but it also seems like a completely separate thing, so I dunno.

Reply:
This can be taken care of separately.